ui/src/app/base/components/node/MachinesHeader/MachinesHeader.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { mount } from "enzyme";
```

Comment:
Could you use the testing library please 🙏

Reply:
I think we should probably have a rule that if you're touching a file you shouldn't have to update to RTL otherwise a lot of PRs are going to get very large.